src/shared/components/downloadControls/DownloadControls.tsx

```diff
@@ -51,6 +51,25 @@ function makeMenuItem(spec:ButtonSpec) {
     );
 }
 
+function openPrintTab(svg: SVGElement) {
+    const mywindow = window.open('', 'Print', 'height=600,width=800');
+    if (mywindow) {
+        // write the element to the new tab
+        mywindow.document.write("<?xml version='1.0'?>"+replaceUnicodeChars((new XMLSerializer()).serializeToString(svg)));    
```

Comment:
it is possible that the issues Tali found are due to the browser handling the xml file differently.   one idea would be to try opening an HTML file with just the SVG inside of it.  if all the browsers can successfully print the current page (with visualization inside layout) but some fail with xml file, then this would be a good route to try.  document.write may work, or look at something called a "data uri", which allows you to set the HTML in the url itself.